.travis.yml

```diff
@@ -37,72 +37,72 @@ matrix:
   - env: TOX_ENV=py36-djmaster-sqlite
     python: 3.6
 
-  - env: TOX_ENV=py27-dj111-postgresql PYTHONIOENCODING=UTF-8
+  - env: TOX_ENV=py27-dj111-postgresql PYTHONIOENCODING=UTF-8 DATABASE_URL=postgres://postgres@localhost/db
```

Comment:
Move `DATABASE_URL` into .travis.yml because cannot run locally (user:password is not same)